engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -1607,7 +1605,7 @@ class IrToTruffle(
                               BadPatternMatch.NonConstantPolyglotSymbol(symbol)
                             )
                           } else {
-                            val value = iop.readMember(polyClass, symbol);
+                            val value = iop.readMember(polyClass, symbol)
```

Comment:
Out of curiosity for our Scala fans: I am surprised the `scalafmtAll` linter isn't set to remove (or fail on) useless `;`...